tests/application_development/libcxx/testcase.yaml

```diff
@@ -1,10 +1,10 @@
 tests:
-  misc.app_dev.libcxx:
+  application_development.libcxx.libcxx:
```

Comment:
To my mind double libcxx look not appropriate `application_development.libcxx.libcxx:`
I will suggest `application_development.libcxx.libcxx_lib:`